pyproject.toml

```diff
@@ -1,3 +1,67 @@
+[build-system]
+requires = ["setuptools>=64"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "nwbinspector"
+version = "0.6.0"
+description = "Tool to inspect NWB files for best practices compliance."
+readme = "README.md"
+authors = [
+    {name = "Cody Baker"},
+    {name = "Steph Prince"},
+    {name = "Szonja Weigl"},
+    {name = "Heberto Mayorquin"},
+    {name = "Paul Adkisson"},
+    {name = "Luiz Tauffer"},
+    {name = "Ben Dichter",  email = "ben.dichter@catalystneuro.com"}
+]
+urls = { "Homepage" = "https://github.com/NeurodataWithoutBorders/nwbinspector" }
+license = {file = "license.txt"}
+keywords = ["nwb"]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: MacOS",
+    "License :: OSI Approved :: BSD License",
+]
+requires-python = ">=3.9"
```

Comment:
This also removes Python 3.8 support, which ends its security upgrades next month